Mathlib/NumberTheory/ArithmeticFunction.lean

```diff
@@ -771,6 +780,19 @@ theorem lcm_apply_mul_gcd_apply [CommMonoidWithZero R] {f : ArithmeticFunction R
     apply Finset.inter_subset_union
   · simp [factorization_lcm hx hy]
 
+theorem map_lcm [CommGroupWithZero R] {f : ArithmeticFunction R}
+    (h_mult : f.IsMultiplicative) {x y : ℕ} (hf : f (x.gcd y) ≠ 0) :
+    f (x.lcm y) = f x * f y / f (x.gcd y) := by
+  rw [←h_mult.lcm_apply_mul_gcd_apply, mul_div_cancel_right₀ _ hf]
+
+theorem eq_zero_of_squarefree_of_dvd_eq_zero [CommMonoidWithZero R] {f : ArithmeticFunction R}
+    (h_mult : IsMultiplicative f) {m n : ℕ} (h_sq : Squarefree n) (hmn : m ∣ n)
```

Comment:
```suggestion
    (hf : IsMultiplicative f) {m n : ℕ} (h_sq : Squarefree n) (hmn : m ∣ n)
```
this file usually uses `hf` for this assumption. 
(Similarly I'd personally use `hn : Squarefree n`, but that's optional)